asm_v_wasm/wasm_bench-d8.py

```diff
@@ -0,0 +1,358 @@
+#!/usr/bin/env python
+#
+# Run wasm benchmarks in various configurations and report the times.
+# Run with -h for help.
+#
+# In the default mode which is "ion+baseline", runs a single shell with
+# --wasm-compiler=baseline and --wasm-compiler=ion and prints three
+# tab-separated columns:
```

Comment:
Can you update this entire comment block too, to make it clear that this is about d8?

Reply:
Done